omnigan/generator.py

```diff
@@ -97,8 +103,16 @@ def __init__(self, opts, latent_shape=None, verbose=None, no_init=False):
             print("  - Created Depth Decoder")
 
         if "s" in opts.tasks and not opts.gen.s.ignore:
```

Comment:
Should we remove the `ignore` parameter? We're not using it anyway and opts.tasks is sufficient to know which tasks we're doing.

Reply:
true, let me change that